infra/prod/westeurope/networking.tf

```diff
@@ -0,0 +1,10 @@
+module "networking" {
+  source = "../_modules/networking"
+
+  location = local.location
+  project  = local.project
+
+  subnet_fims_cidr = "10.0.18.0/26"
```

Comment:
how do you choose this cidr? can you add a comment?

Reply:
The subnet already exixsts, I have actually imported it from `io-infra`.
I believe it is a simple progressive enumeration

Reply:
I'd add a reference to that one anyway.

Reply:
here you go #12